### PR TITLE
[TA] Removed length property in model classes of 5.1.0-beta.2

### PR DESCRIPTION
**Pull request by mssfang**
Currently we don't expose the encoding to the user, so the `length` property can be calculated from the Text property by doing `text.length()`. Because of that we decided to remove `length` until we really need it.

- Removed property `length` from `CategorizedEntity`, `SentenceSentiment`, `LinkedEntityMatch`, `AspectSentiment`, 
`OpinionSentiment`, and `PiiEntity`.

fixes: #15779

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Release History
 ## 5.1.0-beta.2 (Unreleased)
-
+### Breaking changes
+- Removed property `length` from `CategorizedEntity`, `SentenceSentiment`, `LinkedEntityMatch`, `AspectSentiment`,
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:4]: Could you also add why we are removing this in the PR description? Its helps in future referencing this.

Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:4]: Updated

Comment (samvaity) [reply sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:4]: Thanks !

**Review by samvaity — APPROVED** [review]

**Review by iscai-msft — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Release History
 ## 5.1.0-beta.2 (Unreleased)
-
+### Breaking changes
+- Removed property `length` from `CategorizedEntity`, `SentenceSentiment`, `LinkedEntityMatch`, `AspectSentiment`, 
+`OpinionSentiment`, and `PiiEntity`.
```
Comment (deyaaeldeen) [inline sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:5]: Perhaps explain briefly why this change was made?

**Review by maririos — APPROVED** [review]
